src/main/java/com/iota/iri/utils/Pair.java

```diff
@@ -2,10 +2,12 @@
 
 /**
  * Created by paul on 4/15/17.
+ * Edited by footloosejava 03/28/2018
```

Comment:
You can remove this and Paul's line above.
Every time you see a line like this it is a leftover or a mistake.

By the way this class will probably be replaced by `Pair` from Apache soon

Reply:
I noticed 2 different Pair in the project ... Apache makes sense.